test/bin/build_images.sh

```diff
@@ -68,14 +68,36 @@ configure_package_sources() {
     export SOURCE_VERSION_BASE
     export CURRENT_RELEASE_VERSION
     export PREVIOUS_RELEASE_VERSION
-    export LATEST_RHOCP_MINOR
 
     # Add our sources. It is OK to run these steps repeatedly, if the
     # details change they are updated in the service.
     title "Expanding package source templates to ${IMAGEDIR}/package-sources"
     mkdir -p "${IMAGEDIR}/package-sources"
+
+    for minor in ${RHOCPS_TO_ENABLE}; do
```

Comment:
Other places we drive things like this based on the presence of templates, rather than a loop in the code. Could we add y-2, y-1, and y templates and use the same approach here?

Reply:
It's doable, but we'll have to explicitly check if rhocp-4.{y-2,y-1,y} are available and create dedicated variables for them.
This was quicker and shorter approach, but if we're not in hurry we can verify the other approach - I'll have a look